packages/components/src/tooltip/index.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import * as Ariakit from '@ariakit/react/tooltip';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { Children } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { TooltipProps } from './types';
+import Shortcut from '../shortcut';
+import { positionToPlacement } from '../popover/utils';
+
+/**
+ * Time over anchor to wait before showing tooltip
+ *
+ * @type {number}
+ */
```

Comment:
```suggestion
/**
 * Time over anchor to wait before showing tooltip
 */
```

Nit: Type annotation unnecessary?